image.yaml

```diff
@@ -73,17 +73,17 @@ modules:
           - name: jboss-eap-modules
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: EAP_737_CR1
+                  ref: EAP_750_CR1
 
           - name: jboss-eap-image
             git:
                   url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: EAP_737_CR1
+                  ref: EAP_750_CR1
```

Comment:
And here too. `EAP_740_CR1` from May 11-th seems to be the latest one in https://github.com/jboss-container-images/jboss-eap-7-image/tags.